addons/l10n_cz/data/tax_report.xml

```diff
@@ -0,0 +1,978 @@
+                                        <field name='date_scope'>strict_range</field>
+                                    </record>
+                                    <record id='l10n_cz_vat_declaration_line_19_output_tax' model='account.report.expression'>
+                                        <field name='label'>output_tax</field>
+                                        <field name='engine'>tax_tags</field>
+                                        <field name='formula'>VAT 12 Tax</field>
+                                        <field name='date_scope'>strict_range</field>
+                                    </record>
+                                </field>
+                            </record>
+                            <record id='l10n_cz_vat_declaration_line_20' model='account.report.line'>
+                                <field name='sequence'>20</field>
+                                <field name='name'>Reduced - line 13</field>
+                                <field name='code'>R13</field>
+                                <field name='expression_ids'>
+                                    <record id='l10n_cz_vat_declaration_line_20_tax_base' model='account.report.expression'>
+                                        <field name='label'>tax_base</field>
+                                        <field name='engine'>tax_tags</field>
+                                        <field name='formula'>VAT 13 Base</field>
+                                        <field name='date_scope'>strict_range</field>
+                                    </record>
+                                    <record id='l10n_cz_vat_declaration_line_20_output_tax' model='account.report.expression'>
+                                        <field name='label'>output_tax</field>
+                                        <field name='engine'>tax_tags</field>
+                                        <field name='formula'>VAT 13 Tax</field>
+                                        <field name='date_scope'>strict_range</field>
+                                    </record>
+                                </field>
+                            </record>
+                        </field>
+                    </record>
+                </field>
+            </record>
+            <record id='l10n_cz_vat_declaration_line_21' model='account.report.line'>
+                <field name='sequence'>21</field>
+                <field name='name'>II. Other supplies and supplies with a place of supply outside the domestic territory with a right to tax deduction</field>
+                <field name='hierarchy_level'>0</field>
+                <field name='children_ids'>
+                    <record id='l10n_cz_vat_declaration_line_22' model='account.report.line'>
+                        <field name='sequence'>22</field>
+                        <field name='name'>Delivery of goods to another Member State (64) - line 20</field>
```

Comment:
Jump from line 13 to 20 ? is the line number really necessary ? same question for each section we have a jump in the line number

Reply:
u mean the sequence number?

Reply:
No i mean the "line - xx" in the name of the line :smile: